exec/cnex/extension.h

```diff
@@ -0,0 +1,39 @@
+#ifndef EXTENSION_H
+#define EXTENSION_H
+
+struct Ne_ParameterList *parameterlist_alloc(int n);
```

Comment:
As mentioned elsewhere, there's no reason for all these functions to appear in this header file. All you need here is an `extern` declaration for the method table.